packages/trader/src/App/Components/Elements/market-countdown-timer.tsx

```diff
@@ -1,8 +1,9 @@
 import classNames from 'classnames';
 import React from 'react';
 import moment from 'moment';
+import 'moment/min/locales';
```

Comment:
We use `Moment.js` library for handling with `Date`. I found out that there is a method in it for translating dates into different language - `moment.locale`, bur for proper work it requires importing available languages for library. Without this line it won't work